### ISSUE TITLE
Consistent use of biolink:correlated_with in TRAPI queries and responses from KPs

### ISSUE THREAD

**Issue by southalln** [NCATSTranslator/Relay#102]
```
{
  "message": {
    "query_graph": {
      "nodes": {
        "n00": {
          "id": "MONDO:0007254"
        },
        "n01": {
          "category": "biolink:Drug"
        }
      },
      "edges": {
        "e00": {
          "predicate": "biolink:correlated_with",
          "subject": "n00",
          "object": "n01"
        }
      }
    }
  }
}

```
e.g. COHD: the issue with how to represent our data as knowledge in Biolink is a common issue with the clinical KPs and one that we’re actively discussing with Matt and Richard. The discussion is on the bigger question of how to represent data and knowledge coming from clinical and contextual KPs, but it also involves the discussion of what predicates and what node identifiers should be used. We’ve made good headway within our small group and are about to seek input from the larger Translator group to make sure that the model Matt and Richard designed will be usable by other KPs and ARAs.
To answer your question more specifically regarding COHD, currently our only supported predicate types are biolink:correlated_with and its parent biolink:related_to (the same information represented in both predicates). Since COHD mines its edges from associations within observational clinical data, biolink:correlated_with is the most specific predicate that we can assert as knowledge, so we won’t have predicates like biolink:has_phenotype or biolink:treats. We could try to infer things like biolink:treats, but currently, there’s no way to distinguish a biolink:treats edge derived from inference from our KP vs a similarly stated biolink:treats edge derived from curated knowledge from another KP. We currently do not have genetic data in COHD.

**Comment by sstemann** [comment]
@karafecho @CaseyTa Noel logged this issue during the relay.  wanted to put it on TCDC's radar.

**Comment by karafecho** [comment]
Well stated, @CaseyTa. @southalln : The same considerations apply to ICEES+, with one exception, namely, the availability of a limited set of SNP data related to asthma, which we hope to expand as part of our use case on asthma and common pulmonary diseases. We will have a lot of genomic data available as part of our DILI use case.

**Comment by sstemann** [comment]
@karafecho do you have a TCDC repo we can transfer this to?

**Comment by karafecho** [comment]
@sstemann : No, we don't have a repo just yet (this is an agenda item for today's kick-off meeting of the committee).

I do have a question, however (and forgive me if this is obvious): What is the specific issue that needs to be resolved? A general ticket regarding our ongoing discussions with the Biolink HELP team? Something specific to biolink:correlated_with?

**Comment by sstemann** [comment]
@karafecho @kadelakun is there a repo we can transfer this issue to for Clinical?
